gax-httpjson/src/main/java/com/google/api/gax/httpjson/ManagedHttpJsonChannel.java

```diff
@@ -72,24 +72,35 @@ private ManagedHttpJsonChannel(
     this.httpTransport = httpTransport == null ? new NetHttpTransport() : httpTransport;
   }
 
-  public <ResponseT, RequestT> ApiFuture<ResponseT> issueFutureUnaryCall(
+  <ResponseT, RequestT> Runnable createRunnable(
```

Comment:
I might prefer to not split this out as a method and keep the implementation inside `issueFutureUnaryCall` - it appears to only be used in one place (the original method).

Reply:
Sure.